### ISSUE TITLE
System Language Mismatch When Pulling Repo

### ISSUE THREAD

**Issue by MichiFr** [abapGit/abapGit#5543]
H!

Just tried to download this repo https://github.com/SAP/code-pal-for-abap.git and when pulling the repo I get this error:
![image](https://user-images.githubusercontent.com/28045390/167373286-1e59706d-cc70-43bb-a68d-39d0e940f884.png)

"Current login language 'DE' does not match main language 'EN'. Select 'Advanced' > 'Open in Main Language' "

When opening the menu, I do not see this mentioned entry:
![image](https://user-images.githubusercontent.com/28045390/167373482-9efe00ea-fbaa-42cd-87b6-10fbe56deca4.png)

So, basically I assume I just can switch the login language to proceed, nevertheless either the menu Advanced is missing an entry or the error message is wrong.

**Comment by christianguenter2** [comment]
This feature is only available if abapGit full repository is installed, as it needs abapGit transaction code. Though error message could be improved for this case. Anyway, as a workaround create abapGit transaction manually and assign abapGit report to it. Then it should work.
